src/test/java/de/tum/in/www1/artemis/config/MetricsBeanTest.java

```diff
@@ -0,0 +1,118 @@
+package de.tum.in.www1.artemis.config;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.time.ZonedDateTime;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import de.tum.in.www1.artemis.AbstractSpringIntegrationBambooBitbucketJiraTest;
+import de.tum.in.www1.artemis.domain.enumeration.ExerciseType;
+import de.tum.in.www1.artemis.domain.enumeration.QuizMode;
+import de.tum.in.www1.artemis.repository.CourseRepository;
+import de.tum.in.www1.artemis.repository.ExerciseRepository;
+import de.tum.in.www1.artemis.service.CourseService;
+import io.micrometer.core.instrument.MeterRegistry;
+
+public class MetricsBeanTest extends AbstractSpringIntegrationBambooBitbucketJiraTest {
```

Comment:
```suggestion
class MetricsBeanTest extends AbstractSpringIntegrationBambooBitbucketJiraTest {
```
JUnit 5 doesn’t need test to be `public` any more.